assets/stylesheets/_animations.scss

```diff
@@ -1,78 +1,42 @@
-@keyframes fade-in {
-	from {
-		opacity: 0;
-	}
-	to {
-		opacity: 1;
-	}
-}
-
-@mixin animate_fade {
-	animation: animate_fade 0.1s ease-out;
+@mixin editor-animation__animate-fade {
+	animation: editor-animation__animate-fade 0.1s ease-out;
 	animation-fill-mode: forwards;
 }
 
-@mixin move_background {
+@mixin editor-animation__move-background {
```

Comment:
This doesn't seem to be used either, I'd consider removing too.

Reply:
There's this test for this specific animation:

```
	it( 'should ignore keyframes', () => {
		const callback = wrap( '.my-namespace' );
		const input = `
		@keyframes editor-animation__move-background {
			from {
				background-position: 0 0;
			}
		}`;
		const output = traverse( input, callback );

		expect( output ).toMatchSnapshot();
	} );
```

I agree it doesn't seem to be used, but I'm also not quite sure either what this test does or what to do with it. Any suggestions?

Reply:
you can just replace it with any animation (defined or not)

Reply:
To clarify, it's only testing that this editor styles wrapping behavior doesn't touch the animations.